clients/python/MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include LICENSE README.rst
+include src/myanmartools/resources/*
+include tests/*
```

Comment:
Do we need to include the tests in the package distribution?

Reply:
If we include them, the user can test directly from the source distribution after `pip download`. I think it is common practice to include them in the source distribution but not in the packages so they are not installed (see [this answer](https://stackoverflow.com/a/36601576/6293961) and [this answer](https://stackoverflow.com/a/32391295/6293961)).

Reply:
Thanks!